daphne/src/aborts.rs

```diff
@@ -282,7 +284,10 @@ impl From<DapError> for DapAbort {
 
 impl From<CodecError> for DapAbort {
     fn from(_e: CodecError) -> Self {
```

Comment:
Since we're  not ignoring this variable anymore :)
```suggestion
    fn from(e: CodecError) -> Self {
```